test/com/google/javascript/jscomp/ProcessEs6ModulesTest.java

```diff
@@ -63,12 +63,12 @@ protected int getNumRepetitions() {
     return 1;
   }
 
-  static void testModules(CompilerTestCase test, String input, String expected) {
+  static void testModules(CompilerTestCase test, String input, String expected,
+      String fileoverview) {
```

Comment:
Why make it a separate string? Why not just have the 'expected' string start with that comment?